numba_dpex/tests/njit_tests/dpnp_ndarray/test_dpnp_ndarray_type.py

```diff
@@ -11,15 +11,15 @@
 from dpnp import ndarray as dpnp_ndarray
 from numba import njit, typeof, types
 
-from numba_dpex.dpnp_ndarray import dpnp_ndarray_Type
+from numba_dpex.core.types.dpnp_ndarray_types import DpnpNdarray
```

Comment:
the module should be named `dpnp_ndarray_type`

Reply:
Done.